src/test/java/org/apache/sysds/test/functions/builtin/BuiltinMulticlassSVMTest.java

```diff
@@ -99,16 +98,16 @@ private void runMSVMTest(boolean sparse, boolean  intercept, int classes, double
 			double sparsity = sparse ? spSparse : spDense;
 			String HOME = SCRIPT_DIR + TEST_DIR;
 			fullDMLScriptName = HOME + TEST_NAME + ".dml";
-			programArgs = new String[]{ "-explain", "-stats",
+			programArgs = new String[]{
 				"-nvargs", "X=" + input("X"), "Y=" + input("Y"), "model=" + output("model"),
-				"inc=" + String.valueOf(intercept).toUpperCase(), "num_classes=" + classes, "eps=" + eps, "lam=" + lambda, "max=" + run};
+				"inc=" + String.valueOf(intercept).toUpperCase(), "eps=" + eps, "lam=" + lambda, "max=" + run};
 
 			fullRScriptName = HOME + TEST_NAME + ".R";
-			rCmd = getRCmd(inputDir(), Boolean.toString(intercept), Integer.toString(classes),  Double.toString(eps),
+			rCmd = getRCmd(inputDir(), Boolean.toString(intercept),  Double.toString(eps),
```

Comment:
The test is currently failing because the removed parameter shifts the output filename from position 7 to 6 but the R script is not updated consistently.

Reply:
thanks, i have been looking around for that.